src/core/include/openvino/op/util/transpose_attr.hpp

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+namespace ov {
+namespace op {
+
+/** \brief Transpose operator inputs.*/
+enum TransposeIn : size_t { ARG, ORDER, IN_COUNT };
+
+/** \brief Transpose operator outputs.*/
+enum TransposeOut : size_t { ARG_T, OUT_COUNT };
```

Comment:
We usually introduce inputs numbering for complex operators as a private field [like so](https://github.com/openvinotoolkit/openvino/blob/master/src/core/include/openvino/op/embedding_segments_sum.hpp#L70) or don't do it. Since Transpose is really simple operator I propose not to introduce it.

Reply:
This is not matter of operator complexity.

The magic number should not be used [good practice rule](https://rules.sonarsource.com/cpp/RSPEC-109)
This improve readability and code maintenance as it is possible to control these values in one place (can define better names if required).